cgi-bin/DW/Media/Base.pm

```diff
@@ -136,7 +136,11 @@ sub visible_to {
     # at this point, if we don't have a remote user, fail
     return 0 unless LJ::isu($other_u);
 
-    # private check.  if it's us, allow, else fail.
+    # private check.  if it's us or an admin, allow, else fail.
+    my $refer       = { DW::Request->get->headers_in }->{Referer};
```

Comment:
What is this syntax `{ foo }` what does that do? (Either TIL or I've forgotten.)